cockroachcloud/client-certs-dedicated.md

```diff
@@ -0,0 +1,403 @@
+---
+title: Manage Certificate Authentication for SQL Clients
+summary: procedures for managing client certificates for dedicated clusters
+toc: true
+docs_area: manage.security
+cloud: true
+---
+
+SQL clients may authenticate to {{ site.data.products.dedicated }} clusters using public key infrastructure (PKI) security certificates as an alternative to authenticating using a username and password or using [Cluster Single Sign-on (SSO) using CockroachDB Cloud Console](cloud-sso-sql.html) or [Cluster Single Sign-on (SSO) using JSON web tokens (JWT)](../{{site.versions["stable"]}}/sso-sql.html). This page describes how to administer the cluster's certificate authority (CA) certificate and authenticate to a cluster using client certificates.
+
+Refer to [Transport Layer Security (TLS) and Public Key Infrastructure (PKI)](../{{site.versions["stable"]}}/security-reference/transport-layer-security.html) for an overview of PKI certificate authentication in general and its use in CockroachDB.
+
+Refer to [Authenticating to {{ site.data.products.db }}](authentication.html) for an overview of authentication in {{ site.data.products.db }}, both at the level of the organization and at the cluster.
+
+{{site.data.alerts.callout_info}}
+{% include_cached feature-phases/limited-access.md %}
+{{site.data.alerts.end}}
+
+## Provision your cluster's PKI hierarchy
```

Comment:
Should we name this something like "Provision PKI hierarchy for SQL authentication in your cluster"? Basically I want to avoid giving an indication that node-level certs could also be managed in cloud.